article.hid2-ub-cicd.ppam24.tex

```diff
@@ -124,6 +124,25 @@ \subsection{Final Analysis and Urban Scale Energy Evaluation}
 
 The final step involves the \textit{UBEM Simulator}, which generates large-scale outputs that summarize the overall energy consumption and environmental impact of buildings on an urban scale. This comprehensive urban scale analysis merges data from the building energy and air quality models to provide a holistic view of urban environmental quality.
 
+In addition to these exports, the simulator condenses, aggregates and saves relevant quantities on the fly to avoid having to manipulate large data volumes after the simulation. These metrics concern weather information, energy consumption and comfort assessment, and are exported in JSON format. The produced files are uploaded to a data management platform via the CI/CD pipeline, who is then in charge of notifying a different workflow in order to render insightful figures that showcase the simulation results.
```

Comment:
add a \cref for the Fig 10 that shows the entire workflow including the comprehensive report at the end

Reply:
perhaps link to the section where we describe the HPCOps workflow and hence the corresponding Figure  10